max-serve-open-webui/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+version: 1.0
+long_title: "Use Open WebUI with MAX Serve for RAG and Web Search"
+short_title: "Use Open WebUI with MAX Serve"
+author: "Ehsan M. Kermani"
+author_image: "author/billw.jpg"
+author_url: "https://www.linkedin.com/in/welense/"
+github_repo: "https://github.com/modular/max-recipes/tree/main/max-serve-open-webui"
+date: "13-02-2025"
+difficulty: "beginner"
+tags:
+  - max-serve
+  - open-webui
+  - llama3
+
+tasks:
+  - magic run app
```

Comment:
We should include a Procfile.clean. Example [here](https://github.com/modular/max-recipes/blob/main/max-serve-continuous-chat/Procfile.clean)